hooks/basic/publish_session.py

```diff
@@ -155,8 +155,8 @@ def publish(self, task_settings, item):
         self._save_session(item.properties.get("path"), item.properties.publish_version, item)
 
         # Store any file/id dependencies
-        item.properties.publish_dependency_paths = self._get_dependency_paths()
-        item.properties.publish_dependency_ids = self._get_dependency_ids()
+        item.properties.publish_dependency_paths = self._get_dependency_paths(task_settings, item)
+        item.properties.publish_dependency_ids = self._get_dependency_ids(task_settings, item)
```

Comment:
Why aren't these in local_properties like the others?

Reply:
Missed them. Thanks for pointing out!